lib/components/util/styledIcon.tsx

```diff
@@ -51,19 +51,19 @@ export const StyledIconWrapper = styled.span<Props>`
   animation: ${(props) => (props.spin ? rotateAnimation : 'none')} 1s linear
     infinite;
   display: ${(props) => (props.spin ? 'inline-block' : 'initial')};
-  ${StyledIconBase} {
+  svg {
     font-size: ${(props) => getFontSize(props.size)};
     height: 1em;
     transform: ${(props) => `
-      ${props.flipHorizontal ? 'scale(-1,1) ' : ''}
-      ${props.rotate90 ? 'rotate(90deg)' : ''}
-      `};
+      ${props.flipHorizontal ? 'scale(-1,1) ' : ''} 
+      ${props.rotate90 ? 'rotate(90deg)' : ''} 
```

Comment:
Remove trailing whitespace.

Reply:
There's no trailing whitespace on my local copy, so there's nothing here for me to remove.